Cargo.toml

```diff
@@ -22,6 +22,9 @@ rust-version = "1.82" # MSRV
 libc = "0.2"
 libbz2-rs-sys = { path = "libbz2-rs-sys/", default-features = false }
 
+[profile.release]
+debug = true
```

Comment:
This is fine to enable for local builds, but I don't think it should be committed.

Reply:
we do have it enabled in zlib btw, but this should do for me locally

```
// .cargo/config.toml
[build-override.profile.release]
debug = true
```